stories/side-navigation/side-navigation.stories.js

```diff
@@ -1254,3 +1254,85 @@ nestedListWithoutLinks.parameters = {
         disable: true
     }
 };
+
+export const nestedListPopover = () => `<div class="fd-popover fd-popover--right" style="margin-bottom: 200px; margin-right: 200px; margin-left: 200px;">
```

Comment:
what are you achieving with the custom css?

Reply:
to keep distance between the two examples under one doc-section, i was using `margin`.
now I have moved that outside from `fd-popover`.